### ISSUE TITLE
About win10 updates - please consider giving more information to the end user when an update takes longer

### ISSUE THREAD

**Issue by rubyFeedback** [MicrosoftDocs/windows-insider#308]
Hey MS-folks out there,

Please excuse my filing here - I tried to find a useful repository about Win10 in general but ... I
could not find out. I found only semi-related URL entries, including:

https://answers.microsoft.com/en-us/windows/forum/all/get-rid-of-feedback-hub-in-favor-of-github/e3c3b00f-42ae-4c58-b000-6dfe95b1769e

And other feedback-hub related entries. Perhaps in the long run it would help simplify to
streamline feedback, aka "one hub to rule them all". I have had VERY good results in
regards to WSL1 and WSL2, so the teams that handle WSL1 and WSL2 are doing a
good job (in my opinion). I have had mixed results or ... well worse ones elsewhere,
but I digress. I mention this here so that a heroic soul could relocate the content of
this issue request. I encourage to close this request to keep things tidy, but please try
to understand the gist of it.

Some background:

- I have been using linux since close to 20 years, so WSL is actually a "natural" add-on.

Additionally I compile on source, from linux.

My laptop runs Win10 and I am quite happy with how things are. I use ruby as the ultimate
"glue" language for literally everything; even GUIs I can now do, albeit admittedly with a
limited scope, via libui and the ruby-libui bindings. Anyway.

Recently I had some scheduled update ... and the laptop froze. On reboot the icon where
some clock was ticking showed for perhaps 3 minutes or so. The problem is - when you
hit that, you don't know what is going on underneath. Perhaps there is an infinite loop? So
what should I then do if there is no way to tell what is going on?

I suppose I could re-install or do some other re-setting stuff, and so on and so forth. All
viable options. But! This is not entirely what my issue request here is.

I'd love to have an option that gives me more information about what is going on, e. g.
WHY it is loading so long. Normally I get a percentage counter for updates, which is
actually more useful (!) than simply a rotating mouse pointer icon ... 

After about 15 minutes everything works ok, so my initial shock is over. But how to know
without experience? I think there should be a better info-policy. I understand that you 
cater for people who may not be tech-savvy, and this part is ok - but the "power-users"
should not be left alone without any information.

So, TL; DR: when doing updates in general consider giving better informations about what
is going on, at all times. This does not neet to be super-verbose, but simply more information
than an icon that rotates for +3 minutes, and users do not know how many more minutes
anything will run, or distinguish it from infinite loops ...

**Comment by yogkumgit** [comment]
@bleblancMSFT Please help us in resolving this issue. Thanks

**Comment by riverar** [comment]
@rubyFeedback This repository covers the documentation and issues related to that, not Windows in general.

I agree with you though and think this great feedback to put into Feedback Hub, under the category `Install and Update` and sub-category `Downloading, installing, and configuring Windows Update`.

@yogkumgit There's nothing actionable here for docs so I recommend closure.
